src/main/kotlin/org/malachite/estella/task/api/TaskController.kt

```diff
@@ -3,107 +3,129 @@ package org.malachite.estella.task.api
 import org.malachite.estella.commons.EStellaHeaders
 import org.malachite.estella.commons.Message
 import org.malachite.estella.commons.OwnResponses
-import org.malachite.estella.commons.PayloadUUID
 import org.malachite.estella.process.domain.TaskDto
 import org.malachite.estella.process.domain.TaskTestCaseDto
-import org.malachite.estella.services.OrganizationService
 import org.malachite.estella.services.TaskService
 import org.springframework.beans.factory.annotation.Autowired
-import org.springframework.http.HttpStatus
 import org.springframework.http.ResponseEntity
 import org.springframework.transaction.annotation.Transactional
 import org.springframework.web.bind.annotation.*
 
 
 @RestController
+@Transactional
```

Comment:
My nie chcieliśmy unikać tej adnotacji?

Reply:
Ze względu na operacje na Lobach nie da się uniknąć Transactional, jedyne co mogę to ograniczyć scope adnotacji do serwisu, ale nwm czy to cokolwiek zmienia

Reply:
Ponoć zmienia, ale nie pamiętam już co dokładnie i jak bardzo XDD

Reply:
Ogólnie faktycznie któryś serwis ma też transactional, więc utrzymałbym chyba tą konwencję że transactional są u nas jak już serwisy. Ale to w sumie do ustalenia.

Reply:
Wydaje mi się, że lepiej było używać transactional przy endpointach. Nie pamiętam dokładnie, jak to leciało, ale adnotacja jest wiązana z proxy, które spring tworzy na klasie. Teraz mamy taką sytuację, że proxy kontrolera ma dojście do proxy serwisu i jeśli na serwisie stawiamy transactional, to transactional tak naprawdę działa / nie działa - w zależności jak mu się zachce. A przynajmniej tak to pamiętam XDD

Reply:
Dopóki Loby nie wychodzą poza adnotację Transactional, to chyba nie powinno być z tym problemu. Zmienię jak zaproponował Radek, jeśli coś się wykrzaczy to dam poziom wyżej